samcli/commands/pipeline/bootstrap/cli.py

```diff
@@ -138,7 +143,7 @@
 @click.option(
     "--oidc-provider",
     help="The name of the CI/CD system that will be used for OIDC permissions",
```

Comment:
Maybe we can say `we currently only support Github Actions, GitLab and Bitbucket`?